### DISCUSSION TITLE
Where can I find the nushell community?

### DISCUSSION THREAD

**Discussion by jatcwang** [nushell/nushell#2788] [Q&A] [answered]
Where's the best place to talk about Nu and to ask questions? Nu's fantastic but sometimes I had to fallback to bash because I couldn't find how to solve a small obstacle in the pipeline I'm building.

My search didn't seem to yield anything on gitter nor matrix, and there doesn't seem to have a `nushell` tag on StackOverflow either. Where is everyone hiding? :)

(OK, I guess I'm posting in one now there's Github Discussions)

**Comment by kubouch** [comment, marked as answer]
Discord :-) (link in README)
